packages/astro-rss/README.md

```diff
@@ -299,25 +144,69 @@ const item = {
 };
 ```
 
-#### `enclosure.url`
+- `enclosure.url` is the URL where the media can be found. If the media is hosted outside of your own domain you must provide a full URL.
+- `enclosure.length` is the size of the file found at the `url` in bytes.
+- `enclosure.type` is the [MIME type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) for the media item found at the `url`.
 
-Type: `string (required)`
+### stylesheet
 
-The URL where the media can be found. If the media is hosted outside of your own domain you must provide a full URL.
+Type: `string (optional)`
 
-#### `enclosure.length`
+An absolute path to an XSL stylesheet in your project. If you don’t have an RSS stylesheet in mind, we recommend the [Pretty Feed v3 default stylesheet](https://github.com/genmon/aboutfeeds/blob/main/tools/pretty-feed-v3.xsl), which you can download from GitHub and save into your project's `public/` directory.
 
-Type: `number (required)`
+### customData
 
-The size of the file found at the `url` in bytes.
+Type: `string (optional)`
 
-#### `enclosure.type`
+A string of valid XML to be injected between your feed's `<description>` and `<item>` tags. This is commonly used to set a language for your feed:
```

Comment:
```suggestion
A string of valid XML to be injected between your feed's `<description>` and `<item>` tags. 

This can be used to pass additional data outside of the standard RSS spec, and is commonly used to set a language for your feed:
```
Maybe just *something* so that it's clear that this is general purpose, and here's *one* specific thing? I also seem to recall people using this field to pass additional frontmatter properties from their markdown posts? If so, that's also something you could specifically mention in passing, as that's kind of cool.